src/utils/dates.ts

```diff
@@ -0,0 +1,34 @@
+import { Time } from "../components/TimeSelector/TimeSelector";
+
+export const getDaysInMonth = (month,year) => {
+    return new Date(year, month, 0).getDate();
+  };
+
+export const get12Hour = (hh) => {
```

Comment:
I usually prefer not putting numbers in variable names, can you make this `getTwelveHour`?

Reply:
Sure, done